build.gradle

```diff
@@ -0,0 +1,33 @@
+buildscript {
```

Comment:
As I remember we don't need a `buildscript` block here. All our plugins are imported without it.